app/va/va_profile/va_profile_client.py

```diff
@@ -1,40 +1,31 @@
 from __future__ import annotations
+from enum import Enum
+from http.client import responses
 
 import iso8601
 import requests
+
+from app.va.identifier import transform_to_fhir_format, OIDS, IdentifierType
 from app.va.va_profile import (
     NoContactInfoException,
     VAProfileNonRetryableException,
     VAProfileRetryableException,
 )
-from app.va.identifier import is_fhir_format, transform_from_fhir_format, transform_to_fhir_format, OIDS, IdentifierType
-from app.va.va_profile.exceptions import VAProfileIDNotFoundException
-from enum import Enum
-from http.client import responses
-from time import monotonic
-from typing import Dict, List, TYPE_CHECKING
+from app.va.va_profile.exceptions import CommunicationItemNotFoundException, VAProfileIDNotFoundException
 
+from typing import Dict, List, TYPE_CHECKING
```

Comment:
Just re-ordering imports

Reply:
yeah I did it just from memory.  I went ahead and let isort take a whack.  It fixed it.

Reply:
Just pointing out that `typing` is a python builtin so it should really be in the first group. I'm not sure why it's not if you used isort. The imports are a mess everywhere though, so it's not something that has to change right now.